lnrpc/rpc.proto

```diff
@@ -3,205 +3,435 @@ syntax = "proto3";
+    relevant to the wallet.
+    */
     rpc GetTransactions (GetTransactionsRequest) returns (TransactionDetails) {
         option (google.api.http) = {
             get: "/v1/transactions"
         };
     }
+
+    /** lncli: `sendcoins`
+    SendCoins executes a request to send coins to a particular address. Unlike
+    SendMany, this RPC call only allows creating a single output at a time.
+    */
     rpc SendCoins (SendCoinsRequest) returns (SendCoinsResponse) {
         option (google.api.http) = {
             post: "/v1/transactions"
             body: "*"
         };
     }
+
+    /**
+    SubscribeTransactions creates a uni-directional stream from the server to
+    the client in which any newly discovered transactions relevant to the
+    wallet are sent over.
+    */
     rpc SubscribeTransactions (GetTransactionsRequest) returns (stream Transaction);
 
+    /** lncli: `sendmany`
+    SendMany handles a request for a transaction create multiple specified
+    outputs in parallel.
+    */
     rpc SendMany (SendManyRequest) returns (SendManyResponse);
 
+    /** lncli: `newaddress`
+    NewAddress creates a new address under control of the local wallet.
+    */
     rpc NewAddress (NewAddressRequest) returns (NewAddressResponse);
+
+    /**
+    NewAddress creates a new witness address under control of the local wallet.
+    */
     rpc NewWitnessAddress (NewWitnessAddressRequest) returns (NewAddressResponse) {
         option (google.api.http) = {
             get: "/v1/newaddress"
         };
     }
 
+    /** lncli: `signmessage`
+    SignMessage signs a message with the resident node's private key. The
+    returned signature string is `zbase32` encoded and pubkey recoverable,
+    meaning that only the message digest and signature are needed for
+    verification.
+    */
     rpc SignMessage (SignMessageRequest) returns (SignMessageResponse);
+
+    /** lncli: `verifymessage`
+    VerifyMessage verifies a signature over a msg. The signature must be
+    zbase32 encoded and signed by an active node in the resident node's
+    channel database. In addition to returning the validity of the signature,
+    VerifyMessage also returns the recovered pubkey from the signature.
+    */
     rpc VerifyMessage (VerifyMessageRequest) returns (VerifyMessageResponse);
 
+    /** lncli: `connect`
+    ConnectPeer attempts to establish a connection to a remote peer. This is at
+    networking level, and is used for communication between nodes. This is
+    distinct from establishing a channel with a peer.
+    */
     rpc ConnectPeer (ConnectPeerRequest) returns (ConnectPeerResponse) {
         option (google.api.http) = {
             post: "/v1/peers"
             body: "*"
         };
     }
 
+    /** lncli: `disconnect`
+    DisconnectPeer attempts to disconnect one peer from another identified by a
+    given pubKey. In the case that we currently ahve a pending or active channel
+    with the target peer, then
```

Comment:
Oh gosh the original description in `rpcserver.go` had this cut off too. I fixed that one too